src/ui/components/Shared/TracksList.react.js

```diff
@@ -329,20 +330,12 @@ export default class TracksList extends Component {
   }
 
   render() {
-    const tracks = [...this.props.tracks];
+    const { tracks, type } = this.props;
 
-    // TODO (y.solovyov | KeitIG): TrackListHeader component?
     return (
-      <div className='tracks-list-container'> {/* there used to be a tabIndex={ 0 }, I don't remember why */}
+      <div className='tracks-list'> {/* there used to be a tabIndex={ 0 }, I don't remember why */}
```

Comment:
> there used to be a tabIndex={ 0 }, I don't remember why

To be able to Tab over the fields?

Reply:
I think this was related to the keyboard event, but I cannot reproduce any wrong behavior.